Basic/ClientDriven/Assets/Scripts/ClientPlayerMove.cs

```diff
@@ -37,6 +37,12 @@ void Awake()
         Cursor.lockState = CursorLockMode.Locked;
         Cursor.visible = false;
         
+        // ThirdPersonController & CharacterController are enabled only on owning clients. Ghost clients have these two 
+        // components disabled, and will enable a CapsuleCollider. Per the CharacterController documentation: 
+        // https://docs.unity3d.com/Manual/CharacterControllers.html, a Character controller can push rigidbody
```

Comment:
oh nice! good to know
@RikuTheFuffs @chrispope hopefully this will address most (?) issues raised by users on character controller usage? I'm not aware of other issues, if there's any other known issues, please let us know

Reply:
Other issues I read about were related to the `PlayerInput` component being enabled by default, which caused its `Actions` field to be initialized with some weird new instance of `Input Action Asset`.
An easy solution to that is to disable the component by default, and to enable it only on the localPlayer

![image](https://user-images.githubusercontent.com/27730963/205088593-5fc66fb9-bdc2-42a7-aefa-5d2ffd12370c.png)

Reply:
That's a comment on proper usage, @RikuTheFuffs? Just checking since that component is disabled on the PlayerArmature_Networked player prefab, and only enabled on spawn for the owning player.

Reply:
Might worth to still note this in our guide, to make sure inputs are handled only by owner clients and not other clients

Reply:
Added [MTT-5190](https://jira.unity3d.com/browse/MTT-5190) as a follow up to the ClientDriven doc.